modules/apps/frontend-js/frontend-js-clay-sample-web/src/main/resources/META-INF/resources/js/App.js

```diff
@@ -12,20 +12,19 @@
  * details.
  */
 
-import ClayAlert from '@clayui/alert';
+//import ClayAlert from '@clayui/alert';
 import React from 'react';
 
-import '../css/main.scss';
+//import '../css/main.scss';
 
 export default () => {
+	Liferay.Webpack.require("frontend-js-web", function(frontendJsWeb) {
+		console.log(frontendJsWeb);
+	});
+
 	return (
 		<div>
-			<ClayAlert title="Info">
-				This widget is used to test out Clay components. Simply add
-				whatever JS you want to App.js and redeploy.
-			</ClayAlert>
-
-			<div className="clay-test-class">This is where your code goes.</div>
+			Holi
```

Comment:
Interesting... 😂

Reply:
Yes :-D

Reply:
I am not sure I like the name `Webpack` being exposed like this, because the tool we use to prepare our bundles is an implementation detail. Even though webpack is doing the heavy lifting, what we're really using here is "webpack + custom wrapping stuff to integrate it", so I'm wondering whether a more generic/abstract name is more appropriate. You know, to avoid possible confusion when people expect this to be literally just vanilla webpack and then they find that it isn't.

I'm not opposed to `Webpack` — I just have my doubts.

Reply:
I think I'll move the method to `Liferay.require` and leave the rest under `Liferay.Webpack` since the rest is an implementation based on webpack so it doesn't make any sense to "generalize" it. 

Benefits of adding `Webpack` before `require` could be that, if we use another implementation in the future we may have another `Liferay.XXX.require` to distinguish it from this one. But in the other hand if we had such implementation we would probably want it to be smart enough to know which implementation to use, so I'd say we can generalize it.